packages/web/components/navbar/index.tsx

```diff
@@ -185,7 +184,7 @@ export const NavBar: FunctionComponent<
       <>
         <div
           className={classNames(
-            "fixed z-[60] flex h-navbar w-[calc(100vw_-_14.58rem)] place-content-between items-center bg-osmoverse-900 px-8 shadow-md lg:gap-5 md:h-navbar-mobile md:w-full md:place-content-start md:px-4",
```

Comment:
you can pass something like `!shadow-none` in the className prop in order to do this, no need to edit the global navbar